libtransact/src/workload/error.rs

```diff
@@ -0,0 +1,55 @@
+use std;
+use std::error::Error as StdError;
+
+use crate::batch::BatchBuildError;
+use crate::transaction::TransactionBuildError;
+
+#[derive(Debug)]
+pub enum WorkloadError {
+    // Returned when an error occurs when using BatchBuilder.
+    BatchBuildError(Box<BatchBuildError>),
```

Comment:
I don't think these need to be Boxed

Reply:
Correct, this is no longer needed. Fixed.